lib/Model/index.ts

```diff
@@ -315,7 +317,7 @@ export class Model<T extends DocumentCarrier = AnyDocument> {
 			setupFlow.push(() => createTable(this));
 		}
 		// Wait for Active
-		if ((this.options.waitForActive || {}).enabled) {
+		if (typeof this.options.waitForActive !== "boolean" && (this.options.waitForActive || {}).enabled) {
```

Comment:
Same as above.

Reply:
I think I can revert this change given the behavior I mentioned in https://github.com/dynamoose/dynamoose/pull/1180#discussion_r631234611. I originally thought there would be an error if it tried to access an `enabled` property on `true`

Reply:
Reverted in 49cca6b3 but I don't think it'll show up on this PR until it's reopened